cmake/macros/macro_deal_ii_query_git_information.cmake

```diff
@@ -80,11 +80,11 @@ MACRO(DEAL_II_QUERY_GIT_INFORMATION)
     # Format options for git log:
     #   %H   - the full sha1 hash of the commit, aka "revision"
     #   %h   - the abbreviated sha1 hash of the commit, aka "shortrev"
-    #   %cd  - the commit date (for which we use the strict iso date format)
+    #   %cd  - the commit date (for which we use the default date format)
     #
 
     EXECUTE_PROCESS(
-       COMMAND ${GIT_EXECUTABLE} log -n 1 --pretty=format:"revision=%H, shortrev=%h, date=%cd" --date=iso-strict
+       COMMAND ${GIT_EXECUTABLE} log -n 1 --pretty=format:"revision=%H, shortrev=%h, date=%cd" --date=default
```

Comment:
Let's not switch to `default` here. The reason for "iso-strict" here is to get a parseable output: `2022-08-04T13:22:13-04:00`.

Default output produces probably a locale dependent string - which is a nightmare.

Would you mind testing #14206 which avoids the option for all git versions older than 2.2.